src/styles/templates/post.module.scss

```diff
@@ -1,7 +1,3 @@
 .post-container {
   grid-gap: var(--space-24);
-
-  article {
```

Comment:
I think this padding is **mandatory** for desktop sized-screens. But should be removed when on less than `900px` I think. Check the behaviour of the `/learn` pages. It should be similar/or the same.